src/inputs/RadioGroupField.tsx

```diff
@@ -19,6 +20,10 @@ export interface RadioFieldOption<K extends string> {
   description?: string | (() => ReactNode);
   /** The undisplayed value, i.e. an id of some sort. */
   value: K;
+  /** Disable only specific options */
+  disabled?: boolean;
```

Comment:
Suggestion: Usually we have a single `disabled` that uses a union type:

```
disabled?: boolean | ReactNode
```

Just avoid things like `disabled=false disabledTooltip=some message` being ambiguous.